### DISCUSSION TITLE
启动api功能还未实现吗

### DISCUSSION THREAD

**Discussion by GengGode** [PriesiaMioShirakana/DragonianVoice#22] [General]
按钮一直是灰色的
![image](https://user-images.githubusercontent.com/47711102/223905676-b85d3460-64ac-4207-a32b-e843f9220c69.png)

**Comment by PriesiaMioShirakana** [comment]
你需要确定你使用的是VC模型而不是TTS模型，目前该API的主要作用是对接SoVits和DiffSvc的实时变声
